README.md

```diff
@@ -99,27 +122,147 @@ sudo usermod -aG docker $USER
          - "8889:8787"
        volumes:
          # mount conda env into exactly the same path as on the host system - some paths are hardcoded in the env.
```

Comment:
```suggestion
         # TODO: check your conda prefix with `which conda` and adjust everywhere needed
         # mount conda env into exactly the same path as on the host system - some paths are hardcoded in the env.
```

Reply:
won't commit, as which conda won't give you the env paths